lint.swift

```diff
@@ -0,0 +1,547 @@
+        nonMatchingExamples: ["// TODO: refactor", "// TODO: not yet implemented", "// TODO: [cg_2020-03-19] not yet implemented"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles]
+    )
+
+    // MARK: IfAsGuard
+    try Lint.checkFileContents(
+        checkInfo: "IfAsGuard: Don't use an if statement to just return – use guard for such cases instead.",
+        regex: #" +if [^\{\n]+\{\s*return\s*[^\}]*\}(?! *else)"#,
+        matchingExamples: [" if x == 5 { return }", " if x == 5 {\n    return nil\n}", " if x == 5 { return 500 }", " if x == 5 { return do(x: 500, y: 200) }"],
+        nonMatchingExamples: [" if x == 5 {\n    let y = 200\n    return y\n}", " if x == 5 { someMethod(x: 500, y: 200) }", " if x == 500 { return } else {"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles]
+    )
+
+    // MARK: LateForceUnwrapping3
+    try Lint.checkFileContents(
+        checkInfo: "LateForceUnwrapping3: Don't use ? first to force unwrap later – directly unwrap within the parantheses.",
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "separator2": #"\?\."#,
+            "callPart3": #"[^\s\?\.]+"#,
+            "separator3": #"\?\."#,
+            "callPart4": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
+        ],
+        matchingExamples: ["let x = (viewModel?.user?.profile?.imagePath)!\n"],
+        nonMatchingExamples: ["call(x: (viewModel?.username)!)", "let x = viewModel!.user!.profile!.imagePath\n"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$callPart1!.$callPart2!.$callPart3!.$callPart4",
+        autoCorrectExamples: [
+            ["before": "let x = (viewModel?.user?.profile?.imagePath)!\n", "after": "let x = viewModel!.user!.profile!.imagePath\n"]
+        ]
+    )
+
+    // MARK: LateForceUnwrapping2
+    try Lint.checkFileContents(
+        checkInfo: "LateForceUnwrapping2: Don't use ? first to force unwrap later – directly unwrap within the parantheses.",
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "separator2": #"\?\."#,
+            "callPart3": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
+        ],
+        matchingExamples: ["call(x: (viewModel?.profile?.username)!)"],
+        nonMatchingExamples: ["let x = (viewModel?.user?.profile?.imagePath)!\n", "let x = viewModel!.profile!.imagePath\n"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$callPart1!.$callPart2!.$callPart3",
+        autoCorrectExamples: [
+            ["before": "let x = (viewModel?.profile?.imagePath)!\n", "after": "let x = viewModel!.profile!.imagePath\n"]
```

Comment:
Codacy found an issue: [Don't use ? first to force unwrap later ? directly unwrap within the parantheses.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)